capstone-backend/src/main/java/com/google/sps/data/Club.java

```diff
@@ -4,31 +4,21 @@
 import java.util.List;
 
 
-public class Club extends BaseEntity {
+public class Club extends Group {
   
-  private String name;
-  private String description;
-  private String announcement;
-  private List<String> posts;
-  private String ownerID;
-  private List<String> memberIDs;
-  private List<String> inviteIDs;
   private String gbookID;
```

Comment:
Maybe we could use a BookList object to store a list gBookIDs

Reply:
I think right now, for simplicity, it's easier to store a single book. In the future, it would be cool to have a booklist stored here that serves as the 'history' of the books the club has read!

Reply:
I like the idea of a history Antonio!